services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -103,12 +103,12 @@ async def create_computation(
         computational_dag = await create_minimal_computational_graph_based_on_selection(
             complete_dag=complete_dag,
             selected_nodes=job.subgraph or [],
-            force_restart=job.force_restart,
+            force_restart=job.force_restart or False,
```

Comment:
logically that would change nothing, right?. Do you intend to cast, instead?
what about
```python
force_restart=bool(job.force_restart)
```

Reply:
no the problem is that it is set as optional. and pylance complains